.github/workflows/helm-cd.yaml

```diff
@@ -3,7 +3,10 @@ name: Release Charts
 on:
   push:
     branches:
-      - master
+    - release*
+    tags:
+    - v1.*
```

Comment:
Are you planning to include future v1 releases or should we focus on v2+ for this?